cobigen-cli/cli/src/main/java/com/devonfw/cobigen/cli/commands/GenerateCommand.java

```diff
@@ -181,6 +198,37 @@ public Integer doAction() throws Exception {
     return new Tuple<>(generationInputs, selectedGenerableArtifacts);
   }
 
+  /**
+   * Opens a repeatable prompt with a yes/no question and sets the output directory to the current user directory.
+   *
+   */
+  private void setOutputPathWithPrompt() {
+
+    Path outputDirectory = Paths.get(System.getProperty("user.dir"));
```

Comment:
Why do you want to generate to the user home folder? Sounds crazy. A default should be the path the user is currently navigated while issuing cobigen cli, i.e. pwd

Reply:
user.dir = working directory (path where JVM was run from)
user.home = home directory
But if pwd is more reliable, I can change it.